fontbakery-check-description.py

```diff
@@ -20,7 +20,7 @@
 import os
 from TargetFont import TargetFont
 from FontBakeryCheckLogger import FontBakeryCheckLogger
-import fontbakery_checks as checks
+import FontBakeryChecks as checks
```

Comment:
hm.. why? This looks like a name one gives to a python class. The convention for python modules is that they start with a lowercase letter.